website/docs/getting-started/for-designers.md

```diff
@@ -163,7 +163,7 @@ Setting a component to a fixed width in Figma is the same as setting a `width` v
 
 Components and patterns unique to your product should be designed and built locally, extending Helios components and foundations.
 
-![Local component patterns](/assets/getting-started/designers/local-component-patterns.png =706x*)
+![Local component patterns](/assets/getting-started/designers/local-component-patterns.png =706x\*)
```

Comment:
[blocker] I don't know if this was done manually or by the page reformatter, but this I _think_ may break the special syntax we're using to declare the image size.

Reply:
Reverted this